vms/vms/views.py

```diff
@@ -0,0 +1,18 @@
+from django.shortcuts import redirect
+from django.views.generic.edit import FormView
+
+from django.core.mail import send_mail
+from django.core.mail.message import BadHeaderError
+from django.http.response import HttpResponse
+
+from vms import settings
+
+def anonymous_required(func):
+    def as_view(request, *args, **kwargs):
```

Comment:
@amruthasangeeth Okay. 

What is this part of code doing for us? This is for the authentication needed for login/logout, right? Can we name it something which is close to the work this is doing for us. 

After this I also see a need of having comments on top of the functions in the view files. Can you please add a docstring on top of this function. We will start from this. I'll check if there are many, in that case I'll open an issue.

I also want to discuss a bit about this mixin. Ping me when you come on Slack, I'm available till 4:55 P.M.